release_notes/v2.4.1.md

```diff
@@ -0,0 +1,144 @@
+v2.4.1 Release Notes - December 17, 2021
+========================================
+
+
+Improvements
+------------
+
+**peer - Chaincode as a service builder delivered with Fabric release artifacts and in fabric-peer docker image**
+
+Starting in v2.0 chaincode can be run as a service by utilizing the external builder pattern for chaincodes.
+Running 'chaincode as a service' has advantages in Kubernetes and other deployment environments since the chaincode can be managed independently rather than requiring the peer to build chaincode images and start chaincode containers at runtime.
+The external builder pattern required you to deliver a builder script or program alongside the peer.
+Starting in v2.4.1 an external builder for 'chaincode as a service' is available in the Fabric release artifacts and is pre-configured with the fabric-peer docker image,
+removing the need to build your own external builder and repackage and configure the peer.
+
+**peer - Gateway service now randomizes endorsement layouts**
+
+For improved load balancing of endorsement requests and fairer workload distribution across organizations,
+the gateway service now randomizes the peers it connects to for every transaction.
+
+**peer - Gateway service concurrency limits**
+
+Add support for setting gateway service concurrency limits, similar to how concurrency limits are
+set for the existing endorser and deliver services.
```

Comment:
delivery?

Reply:
It actually is called the "deliver service" e.g. at https://hyperledger-fabric.readthedocs.io/en/latest/peer_event_services.html#available-services